simulator.repos

```diff
@@ -2,4 +2,4 @@ repositories:
   simulator/scenario_simulator:
     type: git
     url: https://github.com/tier4/scenario_simulator_v2.git
-    version: master
+    version: RJD-736/autoware_msgs_support  # switch back to master once this branch is merged
```

Comment:
```suggestion
    version: RJD-736/autoware_msgs_support  # TODO(mitsudome-r): switch back to master once this branch is merged
```

It would be better to search TODOs.